lib/ansible/modules/cloud/amazon/elb_target_group.py

```diff
@@ -833,6 +833,7 @@ def main():
                               required_if=[
                                   ['target_type', 'instance', ['protocol', 'port', 'vpc_id']],
                                   ['target_type', 'ip', ['protocol', 'port', 'vpc_id']],
+                                  ['state', 'absent', ['name']],
```

Comment:
This won't change the module's behavior. `name` is already required, so also requiring it when `state == 'absent'` has no effect.

Reply:
ups, I've pushed the wrong changes. My mistake.